app/controllers/oaisys/application_controller.rb

```diff
@@ -1,5 +1,5 @@
 module Oaisys
-  class ApplicationController < ActionController::API
+  class ApplicationController < ActionController::Base
```

Comment:
Maybe just throw a TODO: comment above this, so we remember to look into that error you ran into.

Reply:
Done